### ISSUE TITLE
Discord webhook support

### ISSUE THREAD

**Issue by Kyiro** [ucarno/ow-league-tokens#9]
I propose adding webhook support to know when the tournament started and when you get the tokens.

**Comment by github59173** [comment]
Go all the way and make it a discord bot 👀

**Comment by Kyiro** [comment]
I think that's a bit too much code to add but it'd be cool

**Comment by github59173** [comment]
I have been playing around with the idea and some code to have a drop in solution for this project. Have a config with discord names tied to the account IDs. You can even go further and have Rich presents for watch status. Would be very nice

**Comment by ucarno** [comment]
Too late for now, maybe next year

**Comment by stowmyy** [comment]
i don't think this is possible any more because the bot now requires sensitive information to work and is more difficult to set up. maybe if the overwatch league website works and distributes tokens again at some point
